build-logic/src/main/java/org/ehcache/build/conventions/JacocoConvention.java

```diff
@@ -27,6 +27,7 @@ public void apply(Project project) {
     project.getTasks().withType(Test.class).configureEach(test -> {
       test.getExtensions().configure(JacocoTaskExtension.class, jacoco -> {
         jacoco.getExcludes().add("org.terracotta.tripwire.*");
+        jacoco.getExcludes().add("org.terracotta.management.*");
```

Comment:
what was the (new) issue with this package ?

Reply:
there is a problem with Jacoco.  This is a workaround.

Reply:
Jacoco didn't like the byte code in management.  Code coverage probably isn't important for that package.